data/Harsha1varthan.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "Harsha varthan",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "A Budding Frontend Developer",
+  "links": [
+    {
+      "name": "On Twitter",
+      "url": "https://twitter.com/harshav67037280",
+      "icon": "FaTwitter"
+    },
+     {
+      "name": "On GitHub",
+      "url": "https://github.com/Harsha1varthan",
+      "icon": "FaGitHub"
```

Comment:
```suggestion
      "icon": "FaGithub"
```
Please update this

Reply:
Thank you Bhai

Reply:
When will I get Merged with the main repo?